doc/lightning-offer.7.md

```diff
@@ -6,15 +6,14 @@ SYNOPSIS
 
 *EXPERIMENTAL_FEATURES only*
 
-**offer** *amount* *description* \[*send_invoice*\] \[*label*\] \[*vendor*\] \[*quantity_min*\] \[*quantity_max*\] \[*absolute_expiry*\] \[*recurrence*\] \[*recurrence_base*\] \[*recurrence_paywindow*\] \[*recurrence_limit*\] \[*refund_for*\] \[*single_use*\]
+**offer** *amount* *description* \[*vendor*\] \[*label*\] \[*quantity_min*\] \[*quantity_max*\] \[*absolute_expiry*\] \[*recurrence*\] \[*recurrence_base*\] \[*recurrence_paywindow*\] \[*recurrence_limit*\] \[*single_use*\]
```

Comment:
Why was `vendor` swapped?

I totally agree btw that `offer` was overloaded. We could make the distinction even clearer by calling an offer and subsequent invoice payment either a `push payment` or a `pull payment` :-)

Reply:
vendor is associated with description (they're both strings the user sees which talk about the item).  Since API isn't fixed it makes sense to reorder them while we rework the args anyway.

Naming these things is really hard!  Currently:

A: offer
B: fetchinvoice, pay.

A: offerout
B: sendinvoice
A: (automatically) pay

Not sure how we'd use push-pull here?

Reply:
Right, sounded better in my head.

A: offerpull (user A is offering to pull some funds from B)
B: fetchinvoice

A: offerpush (user A is offering to push some funds to B)
B: sendinvoice
A: pay

But we can always add aliasesat a later stage.